perf-measures/bundle-check/scripts/check-bundle-size.ts

```diff
@@ -19,12 +19,21 @@ async function main() {
 
     const bundleSize = fs.statSync(tempFilePath).size
     const metrics = []
+
     metrics.push({
-      key: 'bundle-size',
-      name: 'Bundle Size',
+      key: 'bundle-size-b',
+      name: 'Bundle Size (B)',
       value: bundleSize,
       unit: 'B',
     })
+
+    metrics.push({
+      key: 'bundle-size-kb',
+      name: 'Bundle Size (KB)',
+      value: (bundleSize / 1024).toFixed(2).toString(),
```

Comment:
```suggestion
      value: parseFloat((bundleSize / 1024).toFixed(2)),
```
It looks like it had to be Number in the custom metrics schema.
https://github.com/k1LoW/octocov/blob/main/report/custom_metrics_schema.json#L51-L53